pep-0684.rst

```diff
@@ -1,16 +1,18 @@
 PEP: 684
 Title: A Per-Interpreter GIL
 Author: Eric Snow <ericsnowcurrently@gmail.com>
-Discussions-To: https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/
+Discussions-To: https://discuss.python.org/t/pep-684-a-per-interpreter-gil/19583
 Status: Draft
 Type: Standards Track
 Content-Type: text/x-rst
 Requires: 683
 Created: 08-Mar-2022
 Python-Version: 3.12
 Post-History: `08-Mar-2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/>`__
+              `29-Sep-2022 <https://discuss.python.org/t/pep-684-a-per-interpreter-gil/19583>`__
```

Comment:
```suggestion
Post-History: `08-Mar-2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/>`__,
              `29-Sep-2022 <https://discuss.python.org/t/pep-684-a-per-interpreter-gil/19583>`__,
```

The first one was missing a comma; I usually just use a comma on the last item too (which both the linter and build system accepts, just like in Python code) to help avoid this mistake